internal/infrastructure/db/migrations/20231113225121_create_template_dd.sql

```diff
@@ -0,0 +1,29 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+CREATE TABLE IF NOT EXISTS template_device_definitions (
+    id BIGSERIAL,
```

Comment:
we normally use KSUID 's for our id columns, we represent that as a char(27)

Reply:
ah but i see we used bigserial for other columns in this project, i didn't catch that before. Let's leave it as is for now.